ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/eventstream/http/services/EventStreamResponseConverter.java

```diff
@@ -88,6 +93,11 @@ private String getResource(Model model, Property predicate) {
 		return stmtOptional.map(statement -> statement.getObject().toString()).orElse(null);
 	}
 
+	private boolean getHasDefaultViewResource(Model model) {
+		return model.listStatements(null, HAS_DEFAULT_VIEW, (Resource) null).nextOptional()
+				.map(statement -> statement.getObject().asLiteral().getBoolean()).orElse(false);
```

Comment:
Since this default view is quite important (Wouter was considering always including it), we might set the default to 'true'? Then it become opt-out instead of opt-in